src/net/http.js

```diff
@@ -66,6 +68,9 @@ Object.assign(pc, function () {
          * Some content types are handled automatically. If postdata is an XML Document, it is handled. If
          * the Content-Type header is set to 'application/json' then the postdata is JSON stringified.
          * Otherwise, by default, the data is sent as form-urlencoded.
+         * @param {Boolean} [options.retryable] If true then if the request fails it will be retried with an exponential backoff.
```

Comment:
Do you think we can just call this "retry"?

Reply:
Oh boy, +1 to that.

Reply:
done